### DISCUSSION TITLE
How to enable skipGeometry parameter?

### DISCUSSION THREAD

**Discussion by codehub-rony** [ldproxy/ldproxy#1231] [Configuration]
How can I enable the skipGeometry parameter for feature data requests?

I get a 400 bad request response stating that the `skipGeometry=true` parameter is invalid. I tried the same [request](https://demo.ldproxy.net/vineyards/collections/vineyards/items?f=json&skipGeometry=true) with vineyards dataset; that works fine.

I couldn't find it in the documentation either.

**Comment by cportele** [comment]
Enable PROJECTIONS

See https://docs.ldproxy.net/services/building-blocks/projections.html
